src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -7,7 +7,7 @@
 
 # pylint: disable=line-too-long
 
-image_long_summary = """                      URN aliases: CentOS, CoreOS, Debian, openSUSE, RHEL, SLES, UbuntuLTS, Win2008SP1, Win2012Datacenter, Win2012R2Datacenter.
+image_long_summary = """                      URN aliases: CentOS, CoreOS, Debian, openSUSE, RHEL, SLES, UbuntuLTS, Win2008R2SP1, Win2012Datacenter, Win2012R2Datacenter.
```

Comment:
The link opened for me. Are you saying it is out of date or inaccurate or that you get error 404?